doc/getting_started.md

```diff
@@ -0,0 +1,36 @@
+# Getting started
+Now that you have successfully installed Azure CLI 2.0 and added the Azure DevOps extension, you are all set to get started!
+
+## Log in to Azure DevOps
+Before you can work with Azure DevOps, you need to log in to Azure CLI using the `az login` command.
+
+If the CLI can open your default browser, it will do so and load a sign-in page.
+Otherwise, you need to open a browser page and follow the instructions on the command line to enter an authorization code after navigating to https://aka.ms/devicelogin in your browser. For more information, please refer [azure cli login page](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest).

```

Comment:
Won't we mention devops login using PAT?

Reply:
I agree. We should mention all login paths . Or redirect to separate sample page about login

Reply:
Since this is tracked in another comment now. Closing this one.